### PR TITLE
Unify config for sc-server and sc-builder

### PR DESCRIPTION
**Pull request by NikitaZotov**
Dependencies:
- Sc-server - https://github.com/ostis-ai/sc-machine/pull/113;
- Sc-server-based sc-web - https://github.com/ostis-ai/sc-web/pull/16;
- Docs for ostis-web-platform - https://github.com/ostis-ai/ostis-web-platform/pull/15.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
+<<<<<<< HEAD
```
Comment (SHtress) [inline README.md:1]: Remove

File: latex/latexmkrc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+sub mpost {
+  my $file = $_[0];
+  my ($name, $path) = fileparse($file);
+  pushd($path);
+  my $return = system "mpost $name";
+  popd();
+  return $return;
+}
+
+$ENV{'TZ'}='Europe/Minsk';
```
Comment (SHtress) [inline latex/latexmkrc:33]: Newline

File: scripts/install_texstudio.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt install texlive-full
+
+sudo add-apt-repository ppa:sunderme/texstudio
+sudo apt remove texstudio-doc texstudio-l10n
+sudo apt update
+sudo apt install texstudio
```
Comment (SHtress) [inline scripts/install_texstudio.sh:9]: Users can use not only texstudio, script probably should download tex deps that you need to build document but you can use for this different tools like texmaker

File: docs/ostis_web_platform.tex [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+К наиболее распространенным программным средствам, ориентированным на хранение и обработку графовых конструкций относятся графовые СУБД (Neo4j \scncite{Neo4j}, ArangoDB \scncite{ArangoDB}, OrientDB \scncite{OrientDB}, Grakn \scncite{Grakn} и др.), а также так называемые rdf-хранилища (Virtuoso \scncite{Virtuoso}, Sesame \scncite{Sesame} и др.), предназначенные для хранения конструкций, представленных в модели RDF. Для доступа к информации, хранимой в рамках таких средств, могут использоваться как языки, реализуемые в рамках конкретного средства (например, язык Cypher в Neo4j), так и языки, являющиеся стандартами для большого числа систем такого класса (например, SPARQL для rdf-хранилищ).
+
+Популярность и развитость такого рода средств приводит к тому, что на первый взгляд целесообразным и эффективным кажется вариант реализации \textit{программного варианта реализации платформы интерпретации sc-моделей} на базе одного из таких средств. Однако, существует ряд причин, по которым было принято решение о реализации \textit{программного варианта реализации платформы интерпретации sc-моделей} с нуля. К ним относятся следующие:
```
Comment (SHtress) [inline docs/ostis_web_platform.tex:8]: Better reformulate sentence "по которым было принято решение о реализации \textit{программного варианта реализации платформы интерпретации sc-моделей} с нуля".

File: docs/ostis_web_platform.tex [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        \item SC-код позволяет описывать также внешние информационные конструкции любого рода (изображения, текстовые файла, аудио- и видеофайлы и т.д.), которые формально трактуются как содержимое \textit{sc-элементов}, являющихся знаками \textit{внешних файлов ostis-системы}. Таким образом, компонентом \textit{программного варианта реализации платформы интерпретации sc-моделей} должна быть реализация файловой памяти, которая позволяет хранить указанные конструкции в каких-либо общепринятых форматах. Реализация такого компонента в рамках современных средств хранения и обработки графовых конструкций также не всегда представляется возможной.
+    \end{scnitemize}
+
+    По совокупности перечисленных причин было принято решение о реализации \textit{программного варианта реализации платформы интерпретации sc-моделей} "с нуля"{} с учетом особенностей хранения и обработки информации в рамках Технологии OSTIS.}
```
Comment (SHtress) [inline docs/ostis_web_platform.tex:17]: Again try not to say "с нуля" say something like our own solution/model/realization

File: docs/ostis_web_platform.tex [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+\scnrelfromset{декомпозиция программной системы}{Программная модель sc-памяти;Реализация интерпретатора sc-моделей пользовательских интерфейсов}
+\scnexplanation{Текущий \textit{Программный вариант реализации платформы интерпретации sc-моделей компьютерных систем} является web-ориентированным, то есть с точки зрения современной архитектуры каждая \mbox{ostis-система} представляет собой web-сайт доступный онлайн посредством обычного браузера. Такой вариант реализации обладает очевидным преимуществом -- доступ к системе возможен из любой точки мира, где есть Интернет, при этом для работы с системой не требуется никакого специализированного программного обеспечения. С другой стороны, такой вариант реализации обеспечивает возможность параллельной работы с системой нескольких пользователей.
+
+В то же время, взаимодействие клиентской и серверной части организовано таким образом, что \mbox{web-интерфейс} может быть легко заменен на настольный или мобильный интерфейс, как универсальный, так и специализированный.
+
+Данный вариант реализации распространяется под open-source лицензией, для хранения исходных текстов используется хостинг Github и коллективная учетная запись ostis-ai.
```
Comment (SHtress) [inline docs/ostis_web_platform.tex:23]: Be careful with license

File: scripts/run_texstudio.sh [hunk 7]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+texstudio ../latex/main.tex
```
Comment (SHtress) [inline scripts/run_texstudio.sh:3]: Are you sure we need this?

**Review by SHtress — CHANGES_REQUESTED** [review]

**Review by SHtress — APPROVED** [review]

**Review by KovalM — APPROVED** [review]

File: scripts/run_scweb.sh [hunk 8]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 cd ../sc-web/server/
-python3 app.py --cfg=../../web-platform-config.ini
+python3 app.py
```
Comment (KovalM) [inline scripts/run_scweb.sh:4]: I propose use run_scweb.sh from sc-web

**Review by KovalM — CHANGES_REQUESTED** [review]

**Review by KovalM — APPROVED** [review]

**Review by SHtress — APPROVED** [review]

**Pull request merged**